components/config/histogram_config.py

```diff
@@ -0,0 +1,23 @@
+import dash_html_components as html
+import dash_bootstrap_components as dbc
+
+from components.config import histogram_column_picker
+from components.config import date_picker
+
+
+def get_component(columns, min_date, max_date, default_end_date):
+    """ Return the configuration component for the histogram view.
+
+    Keyword arguments:
+    columns -- The available columns in the uploaded data
+    min_date -- The minimum date allowed to pick with the datepicker
+    max_date -- The maximum date allowed to pick with the datepicker
+    default_end_date -- The default end date for the datepicker. The
+        default start date is the `min_date`.
+    """
+
+    return html.Div([
+        date_picker.get_component(min_date, max_date, default_end_date),
+        histogram_column_picker.get_component(columns),
+        dbc.Button('Apply', id='apply', outline=True, color='success')
+    ])
```

Comment:
I think it might be nice to have a configuration for the number of bins used, too. Currently the number is quite low for inspecting the distribution of magnitudes, for example

Reply:
So, I couldn't figure out a way to choose an exact number of bins. Setting this nbinsx (https://plotly.com/javascript/reference/#histogram-nbinsx) to a value, it is possible to set up a maximum bin size, and the biggest optimal bin size (smaller than the maximum) is chosen. This works, if the exact number of bins is not an issue, and rather orders of magnitude of bins.

By testing the values, the bin sizes are usually selected to be "nice" round numbers like 0.5 and 0.2 etc. I added this configuration to the branch, but we can choose if this is acceptable.

Reply:
I think it's good as is, at least there is some control over the number of bins, and the label clearly states it is for selecting the maximum number of bins